gym/spaces/tuple.py

```diff
@@ -9,19 +10,30 @@
 
 
 class Tuple(Space[tuple], Sequence):
-    """
-    A tuple (i.e., product) of simpler spaces
+    """A tuple (more precisely the cartesian product) of simpler spaces.
```

Comment:
"simpler spaces" is not necessarily true, as it is possible to have a Tuple of Tuple of Discrete Spaces

Reply:
I agree and this term also shows up in older docstrings. What about "constituent spaces"?

Reply:
That would work or `Subspaces` as Tuples variable is called `subspaces`